Description: Add missing header include
 In gcc 4.7, many of the standard C++ library include files have been
 edited to no longer include <unistd.h> to remove namespace pollution.
 The missing include is added to fix FTBFS with gcc 4.7.
Author: Tobias Winchen <winchen@physik.rwth-aachen.de>
Last-Update: 2012-05-17


--- a/src/pointmatch.cpp
+++ b/src/pointmatch.cpp
@@ -25,6 +25,7 @@
 
 #include <math.h>
 #include <stdlib.h>
+#include <unistd.h>
 
 #include "main.h"
 #include "mmsubs.h"

